react/src/views/BoxEdit/components/BoxEdit.tsx

```diff
@@ -195,14 +196,21 @@ function BoxEdit({
   }, [productId, productAndSizesData, boxData, resetField]);
 
   if (boxData == null) {
-    return <Box>No data found for a box with this id</Box>;
+    notificationVar({
```

Comment:
Does it make sense to have this check right away at the start of the function in line 108 or am I missing something?